Lua/Comments.tmPreferences

```diff
@@ -15,6 +15,18 @@
 				<key>value</key>
 				<string>-- </string>
 			</dict>
+			<dict>
+				<key>name</key>
+				<string>TM_COMMENT_START_2</string>
+				<key>value</key>
+				<string>--[[</string>
+			</dict>
+			<dict>
+				<key>name</key>
+				<string>TM_COMMENT_END_2</string>
+				<key>value</key>
+				<string>--]]</string>
```

Comment:
IMO it should be obvious from the color scheme and the scoping that a comment block ends.